brainrender/actors/neurons.py

```diff
@@ -50,6 +51,7 @@ def __init__(
         :param color: str,
         :param neuron_radius: float, radius of axon/dendrites
         :param soma_radius: float, radius of soma
+        :param invert_dims: bool, invert dimensions of neuron
```

Comment:
Could this docstring be expanded? It's not immediately obvious what "inverting dimensions" means.